sdkproject/Assets/Mapbox/Core/mapbox-sdk-cs/Map/TileErrorEventArgs.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Mapbox.Map
+{
+	using System;
+	using Mapbox.Unity.MeshGeneration.Data;
+	using System.Collections.Generic;
+
+	public class TileErrorEventArgs {
```

Comment:
Does it make sense to include type type, as well? If a tile request fails, how would we issue a specific request again?

Reply:
Done!